### DISCUSSION TITLE
Nod - Ubuntu

### DISCUSSION THREAD

**Discussion by IonutCo** [CoopNetwork/GMD-blockchain#10] [Q&A] [answered]
Salut,

Va rog sa ma ajutati cu urmatoarele informatii:

1. Ubuntu Desktop poate rula pe MacOS 2020?
2. Avand in vedere ca nodul trebuie sa fie online permanent, acesta trebuie sa ruleze pe un VPS?

Multumesc frumos!

**Comment by georgeasiminei** [comment, marked as answer]
Salutare,

1. Da, pe Mac poti rula masina virtuala iar pe masina virtuala poti instala orice. Dar nu cred ca e necesar. Poti rula nodul direct pe mac folosind scripturile de linux.
2. Daca vrei sa faci forging, da e bine sa tii nodul online permanent, daca nu, nu e neaparata nevoie. Daca e online permanent, sigur, poti rula pe un VPS.
